From afe9094962847ff81cb85ed5e84bfa1a852a1ba6 Mon Sep 17 00:00:00 2001
From: Anson Huang <Anson.Huang@nxp.com>
Date: Tue, 6 Jun 2017 00:15:41 +0800
Subject: [PATCH 1849/5242] MLK-15008-2 ARM64: dts: freescale: imx8qm: update
 cpu-freq opp table

commit  c5973d8a93b68edc763b0b3f8dc906b294da3683 from
https://source.codeaurora.org/external/imx/linux-imx.git

Update i.MX8QM cpu-freq opp table according to SCFW setting,
and move the cpu-freq opp table to soc dtsi instead of board
dtb, as it is SoC feature.

Signed-off-by: Anson Huang <Anson.Huang@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts  |   16 ----------------
 arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi      |   16 ++++++++++++++++
 2 files changed, 16 insertions(+), 16 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
index 22e2789..dffac3f 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm-lpddr4-arm2.dts
@@ -471,22 +471,6 @@
 	status = "okay";
 };
 
-&A53_0 {
-	operating-points = <
-		/* kHz    uV */
-		1296000 1150000
-	>;
-	clocks = <&clk IMX8QM_A53_DIV>;
-};
-
-&A72_0 {
-	operating-points = <
-		/* kHz    uV */
-		1800000 1150000
-	>;
-	clocks = <&clk IMX8QM_A72_DIV>;
-};
-
 &gpu_3d0 {
 	status = "okay";
 };
diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
index 59b0528..ecd7c31 100644
--- a/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8qm.dtsi
@@ -1639,3 +1639,19 @@
 		status = "disabled";
 	};
 };
+
+&A53_0 {
+	operating-points = <
+		/* kHz    uV */
+		1200000 1150000
+	>;
+	clocks = <&clk IMX8QM_A53_DIV>;
+};
+
+&A72_0 {
+	operating-points = <
+		/* kHz    uV */
+		1584000 1150000
+	>;
+	clocks = <&clk IMX8QM_A72_DIV>;
+};
-- 
1.7.9.5

